### DISCUSSION TITLE
How can I install and play with notebook7?

### DISCUSSION THREAD

**Discussion by startakovsky** [jupyter/notebook#6903] [Q&A] [answered]
I would like to know the steps to install notebook 7 and then work with this pull request: https://github.com/jupyterlab/jupyterlab/pull/14102

Is there an easy way to do this or a document you can point me to?

**Comment by startakovsky** [comment]
I read this but the RetroLab repository looks archived? (cc: @jtpio)


From https://jupyter.org/enhancement-proposals/79-notebook-v7/notebook-v7.html: 

This document proposes that the next major release of the [Jupyter Notebook](https://jupyter-notebook.readthedocs.io/) application, version 7, will be based on the JupyterLab codebase, but will provide an equivalent user experience to the current (version 6) application. Jupyter Notebook version 7 (“Notebook v7” henceforth) will achieve this by using the current version of [RetroLab](https://github.com/jupyterlab/retrolab). The following three key ideas are central to this planned transition:

**Reply by startakovsky** [reply]
`pip install notebook` yields `Successfully installed notebook-6.5.4` which is not `7`.

**Reply by jtpio** [reply, marked as answer]
You would need to specify `--pre` (for pre-release) when installing notebook: `pip install --pre notebook`.

**Reply by tinosai** [reply]
> You would need to specify `--pre` (for pre-release) when installing notebook: `pip install --pre notebook`.

Would it be possible to write it somewhere on the Jupyter website? It's really hard to come across this info.

**Reply by jtpio** [reply]
@tinosai  Notebook 7 is now available as a stable release, so a plain `pip install notebook` should be enough to install it.
